api/client/javascript/src/client/schemas.ts

```diff
@@ -6050,6 +6050,34 @@ export interface components {
       | 'invalid_scope'
       | 'server_error'
       | 'temporarily_unavailable'
+    /** @description Package price with spend commitments. */
+    PackagePriceWithCommitments: {
+      /**
+       * @description The type of the price. (enum property replaced by openapi-typescript)
+       * @enum {string}
+       */
+      type: 'package'
+      /**
+       * Amount
+       * @description The price of one package.
+       */
+      amount: components['schemas']['Numeric']
```

Comment:
This should be type `Money` (see https://github.com/openmeterio/openmeter/blob/main/api/spec/src/productcatalog/prices.tsp#L68)

Reply:
let's continue here: https://github.com/openmeterio/openmeter/pull/2494